common/src/main/java/com/github/teamfusion/spyglassplus/mixin/EntityMixin.java

```diff
@@ -0,0 +1,84 @@
+package com.github.teamfusion.spyglassplus.mixin;
+
+import com.github.teamfusion.spyglassplus.enchantment.SpyglassPlusEnchantments;
+import com.github.teamfusion.spyglassplus.entity.ScopingEntity;
+import com.github.teamfusion.spyglassplus.util.CommonPlayerLookup;
+import com.github.teamfusion.spyglassplus.world.SpyglassRaycasting;
+import dev.architectury.networking.NetworkManager;
+import io.netty.buffer.Unpooled;
+import net.minecraft.enchantment.EnchantmentHelper;
+import net.minecraft.entity.Entity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.network.PacketByteBuf;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.Collection;
+import java.util.Collections;
+
+import static com.github.teamfusion.spyglassplus.network.SpyglassPlusNetworking.*;
+
+@Mixin(Entity.class)
+public class EntityMixin {
+    @Shadow public World world;
+    @Unique private int lastIndicateEntityId = -1;
+
+    /**
+     * Tracks and updates entities this is indicating.
+     */
+    @Inject(method = "tick", at = @At("TAIL"))
+    private void tickIndicate(CallbackInfo ci) {
+        if (this.world.isClient) return;
+
+        Entity that = (Entity) (Object) this;
+        if (that instanceof ScopingEntity scoping) {
+            if (scoping.isScoping()) {
+                ItemStack stack = scoping.getScopingStack();
+                if (EnchantmentHelper.getLevel(SpyglassPlusEnchantments.INDICATE.get(), stack) > 0) this.raycastAndSendIndicateUpdate(CommonPlayerLookup.tracking(that));
+            } else {
+                if (this.lastIndicateEntityId != -1) this.sendIndicateUpdate(CommonPlayerLookup.tracking(that), -1);
```

Comment:
Changing one-lined if's into block if's would be great for clarity and readability